cwd_saml_mapping.module

```diff
@@ -220,7 +220,8 @@ function cwd_saml_mapping_preprocess_item_list(&$variables) {
 function cwd_saml_mapping_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
   if ($form_id == 'user_login_form') {
     $config = \Drupal::config('cwd_saml_mapping.config_form');
-
+    $form['#cache'] = ['max-age' => 0];
+    
```

Comment:
This might not be enough. You might also need to add Cache::invalidateTags() before loading.